### ISSUE TITLE
different sizes of the same image lead to different results

### ISSUE THREAD

**Issue by umiiwa** [matterport/Mask_RCNN#1725]
When I resize the image to (1033, 688) or (1031,688), I will get the correct result as below.
![image](https://user-images.githubusercontent.com/50259117/64220565-68646080-cefc-11e9-9654-b71609c81163.png)
But  when I resize the image to (1032, 688), the truck disappeared. 
![image](https://user-images.githubusercontent.com/50259117/64220647-abbecf00-cefc-11e9-88e1-3f4a687d984d.png)
Why?

**Comment by korosigSZTE** [comment]
Your model learned wrong features (example the size of the car) and overfitted (?) for your training/validation set. 
To avoid this, you could use:
- more pictures for training
- turn on Mask RCNN augment/augmentation option
```
    def train(self, train_dataset, val_dataset, learning_rate, epochs, layers,
              augmentation=None, custom_callbacks=None, no_augmentation_sources=None)

augmentation: Optional. An imgaug (https://github.com/aleju/imgaug)
            augmentation. For example, passing imgaug.augmenters.Fliplr(0.5)
            flips images right/left 50% of the time. You can pass complex
            augmentations as well. This augmentation applies 50% of the
            time, and when it does it flips images right/left half the time
            and adds a Gaussian blur with a random sigma in range 0 to 5.
                augmentation = imgaug.augmenters.Sometimes(0.5, [
                    imgaug.augmenters.Fliplr(0.5),
                    imgaug.augmenters.GaussianBlur(sigma=(0.0, 5.0))
                ])
```

- or you could use python imgaug package to create more pictures
https://github.com/aleju/imgaug

If you are not familiar with image augmentation:
https://towardsdatascience.com/data-augmentation-and-images-7aca9bd0dbe8

**Comment by May-forever** [comment]
> When I resize the image to (1033, 688) or (1031,688), I will get the correct result as below.
> ![image](https://user-images.githubusercontent.com/50259117/64220565-68646080-cefc-11e9-9654-b71609c81163.png)
> But when I resize the image to (1032, 688), the truck disappeared.
> ![image](https://user-images.githubusercontent.com/50259117/64220647-abbecf00-cefc-11e9-88e1-3f4a687d984d.png)
> Why?

Hi umiiwa, 

I want to use Mask-RCNN to train on my custom dataset (just one class).

First of all, I use Labelme to create groundtruth for each image. Each image has a .json file as its ground-truth.

After that, I don't know what should I do...

I want to know how to convert my groundtruth file to a format that Mask-rcnn can use. Do you have any tools ?

Looking forward to hearing from you, thanks in advance.

With best regards,
Ming
